index.bs

```diff
@@ -404,6 +406,9 @@ like
                 { preventClose, preventAbort, preventCancel, signal } = {})
     <a href="#rs-pipe-to">pipeTo</a>(dest, { preventClose, preventAbort, preventCancel, signal } = {})
     <a href="#rs-tee">tee</a>()
+
+    <a href="#rs-get-iterator">getIterator</a>({ preventCancel } = {})
```

Comment:
What sort of order are we trying to maintain inside these class definitions?

For now, I've put them in the order as they appear in the spec text. If we're aiming for "constructor, properties, methods, symbols", then `getIterator` should go before `getReader`... but should we then also move the spec text for `getIterator` before that of `getReader`? 🤷‍♂️

Reply:
I think yes, it has to go before getReader, and yes, that means moving the spec text. Sorry :smile:.

Reply:
Fair enough. 😛